src/main/java/teammates/ui/template/StudentFeedbackResultsQuestionWithResponses.java

```diff
@@ -1,7 +1,13 @@
 package teammates.ui.template;
 
 import java.util.List;
+import java.util.stream.Collectors;
 
+/**
+ * View Model for student/feedbackResults/questionWithResponses.tag
+ * Manages student feedback results questions details, self-responses tables and others-responses tables separately
+ * for a specific question.
+ */
 public class StudentFeedbackResultsQuestionWithResponses {
```

Comment:
```
/**
* View Model for student/feedbackResults/questionWithResponses.tag.
*
* Manages student feedback results questions details, self-responses tables and others-responses tables separately
* for a specific question.
*/
```
The first line is the summary and should ends with `.`.
There should be a empty space between the summary and the detail description.

Reply:
As specified by the checkstyle, do I need to put a `<p>` before the description?